ops/github-webhook/deploy-from-github.sh

```
@@ -0,0 +1,27 @@
+exec 9>"$LOCK_FILE"
+flock -n 9 || exit 0
+
+cd "$APP_DIR"
+git fetch --prune origin "$BRANCH"
+git checkout -B "$BRANCH" "origin/$BRANCH"
+git reset --hard "origin/$BRANCH"
+
+docker compose -f "$COMPOSE_FILE" build --pull erp-backend frontend
+docker compose -f "$COMPOSE_FILE" up -d --remove-orphans
+
+for i in {1..30}; do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)